realtime/src/main/java/com/mx/path/service/connection/realtime/apis/AccountsAPI.java

```diff
@@ -0,0 +1,64 @@
+package com.mx.path.service.connection.realtime.apis;
+
+import lombok.Data;
+import lombok.Getter;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+import com.mx.common.accessors.UnauthorizedException;
+import com.mx.common.http.HttpStatus;
+import com.mx.common.http.MediaType;
+import com.mx.path.service.connection.realtime.MdxRealtimeConnection;
+import com.mx.path.service.connection.realtime.models.MdxAccount;
+
+public class AccountsAPI {
```

Comment:
I think we decided that we would call these connections as well. They are classified as "sub-connections".